net/netifyd/Makefile

```diff
@@ -51,10 +60,12 @@ define Package/netifyd/conffiles
 /etc/netify.d/site.uuid
 endef
 
-TARGET_CFLAGS+=-Wno-psabi
-TARGET_CXXFLAGS+=-std=c++11 -Wno-psabi
+TARGET_CFLAGS+=-ffunction-sections -fdata-sections -Wno-psabi
+TARGET_CXXFLAGS+=-std=c++11 -ffunction-sections -fdata-sections -Wno-psabi
```

Comment:
This is added by `PKG_BUILD_FLAGS:=gc-sections` already.

Reply:
Thanks, fixed.